### DISCUSSION TITLE
最后一个基于php的较完整版本

### DISCUSSION THREAD

**Discussion by M1oFeather** [baimianxiao/AcgDraw#5] [Announcements]
由于php的种种限制，该项目将使用python进行重写
此版本较稳定

<hr /><em>This discussion was created from the release <a href='https://github.com/baimianxiao/Arknights-Draw/releases/tag/v2.0.2'>最后一个基于php的较完整版本</a>.</em>

**Comment by sahuidhsu** [comment]
好！那后续这个版本还会继续更新卡池信息吗QAQ

**Reply by sahuidhsu** [reply]
> 好！那后续这个版本还会继续更新卡池信息吗QAQ

因为我现版本使用admin.html没法正常更新了，报error，浏览器控制台显示的错误是Failed to load resource: 错误的URL
导致现在进行某些特定卡池的抽卡时会有些干员立绘不显示。。

**Reply by M1oFeather** [reply]
1.如果您使用的是readme.md提供的API地址，那么卡池数据，干员数据以及功能都会保持最新状态（PS：地址可能更新，旧的地址会自动重定向到新地址，不过还是建议更新到新地址。且更新可能存在延迟，具体视wiki数据更新情况）
2.如果您使用的是自行部署的API，在无法正常使用admin.html进行一键更新时，可以直接访问include/admin.php进行更新，耐心等待后返回200即为更新成功
3.关于部分干员寻访图缺失，此情况为v2.0.1beta遗留bug，起因是在wiki还未更新数据时所以一键更新会将空链接保存到数据文件，导致干员图片未下载。解决方法：在请确保您的API文件均为最新版本时，删除data目录下有关character的所有文件，然后进行一键更新。如果仍未解决，请重新部署API
4.虽然php版本的绝大部分功能可以正常运行，但是语法逻辑，更新逻辑以及运行效率仍然存在较多问题，可能在运行中遇到未知bug，所以在未来python重写版本更新后，我希望您能更新新的版本以获得最佳的体验，当然我会保证其部署方法尽可能的简单。
希望能解决你的问题，感谢支持

**Reply by sahuidhsu** [reply]
感谢解答！往后写python版本我也会继续支持的！
我尝试了删除所有character信息重新一键更新，结果出现了timeout报错。。大概是因为我服务器在香港，连PRTS太慢？
那我这边先使用您的API吧，暂时先不用自己部署的了

**Reply by M1oFeather** [reply]
未来会考虑非大陆服务器的更新问题
